### ISSUE TITLE
Documentation refers to non existing commands

### ISSUE THREAD

**Issue by honzakral** [elastic/elastic-agent#2400]
For confirmed bugs, please report:
- Version: 8.6.2

in https://www.elastic.co/guide/en/fleet/current/debug-standalone-agents.html#inspect-configuration there is a reference to `elastic-agent inspect output` command which doesn't work for me:

```
# elastic-agent inspect output             
Error: accepts 0 arg(s), received 1
Usage:
  elastic-agent inspect [flags]
  elastic-agent inspect [command]
....
```

Version info:
```
Binary: 8.6.2 (build: 913c02bea9b13dec4d5c5f3057b5b397344e3298 at 2023-02-13 16:51:45 +0000 UTC)
Daemon: 8.6.2 (build: 913c02bea9b13dec4d5c5f3057b5b397344e3298 at 2023-02-13 16:51:45 +0000 UTC)
```

**Comment by honzakral** [comment]
another similar issue is that `--pprof` is not a recognized flag for `diagnostics` command, neither is the `collect` subcommand as mentioned in https://www.elastic.co/guide/en/fleet/current/fleet-troubleshooting.html#_collect_elastic_agent_diagnostics_bundle

**Comment by cmacknz** [comment]
The diagnostics command and structure changed in 8.6, we missed updating a few pages. The command is now `elastic-agent diagnostics`, the `collect` sub-command was removed. The pprof profiles are also collected by default now so the `--pprof` argument was removed.

`elastic-agent inspect output` was also removed. `elastic-agent inspect` will output the current agent policy, while `elastic-agent inspect --variables` will show the configuration with all variables and conditions applied.

I've updated the docs in https://github.com/elastic/observability-docs/pull/2808 to remove the obsolete commands and arguments.
